### PR TITLE
Add  Accesibility

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
- Page titles- No issues found
- Image text alternatives - All images have alternative texts ( No issues found)
- Text heading - No issues found
- Color contrast - No issues found
- Resize- No issues found
- Interaction- No issues found
- Moving content - No issues found
- Multimedia - No issues found
- The basic structure of the page - No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,7 +176,7 @@ <h4 class="about-me-title">About me</h4>
 
                 <div class="card framework-card">
                     <div class="framework card-img">
-                        <img src="pics/framework-icon.png" alt="icon">
+                        <img src="pics/framework-icon.png" alt="icon framework">
                     </div>
                     <div class="card-details">
```
Comment (micheaol) [inline index.html:181]: **Resize Text**:
* Please ensure that Text, images, and other content do not overlap.
Kindly fix that.

image

File: index.html [hunk 2]
```diff
@@ -176,7 +176,7 @@ <h4 class="about-me-title">About me</h4>
 
                 <div class="card framework-card">
```
Comment (micheaol) [inline index.html:177]: **Contrast ratio ("color contrast")**:

* Please ensure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text.
Please check the report [HERE](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2022-03-01-at-08-45-54-877?overview)

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @MICHAELMUNAVU83 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @MICHAELMUNAVU83 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**